packages/graph/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "graph"
```

Comment:
Perhaps for now lets call it `hash-graph` and then come up with a better name later on if we need to